projects/avataaars-angular/src/lib/components/avatar/avatar.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AvatarStyle} from '../../models/enums/avatarStyle.enum';
+
+@Component({
+  selector: 'avatar',
+  templateUrl:'./avatar.component.html',
+  styles: [
+  ]
+})
+export class AvatarComponent implements OnInit {
+  @Input() style: object ={};
+  @Input() avatarStyle = AvatarStyle.Circle;
+  
+  constructor() {
+   
```

Comment:
lets get rid of unnecessary white space lines.

Reply:
Done